docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,273 @@
+.. _scheduling_tutorial:
+
+***************************
+Scheduling an Observing Run
+***************************
+
+.. note::
+
+    Some terms used have been defined in :ref:`Terminology <terminology>`.
+
+Contents
+========
+
+* :ref:`scheduling-defining_targets`
+
+* :ref:`scheduling-creating_constraints_and_observing_blocks`
+
+* :ref:`scheduling-creating_a_transitioner`
+
+* :ref:`scheduling-scheduling`
+
+.. _scheduling-defining_targets:
+
+Defining Targets
+================
+
+Say we want to observe Vega, Arcturus, Deneb, and Polaris, as well as a few
+other stars in Ursa Minor. For this we will be using the Apache Point Observatory.
+
+First we define our `~astroplan.Observer` object (where we are observing from):
+
+.. code-block:: python
+
+    >>> from astroplan import Observer
+
+    >>> apo = Observer.at_site('apo)
```

Comment:
Don't forget to build locally with both Python 2 and 3 before pushing to GitHub to catch things like this, with 
```
python setup.py build_sphinx -lo
```